dataactbroker/app.py

```diff
@@ -103,7 +83,7 @@ def runApp():
 
 if __name__ == '__main__':
     runApp()
-    proc = multiprocessing.Process(target=checkDynamo)
+    proc = multiprocessing.Process()
```

Comment:
Agreed that this can be removed; no need for a separate process anymore (though it might not have been hit while the app was running anyway?)